### PR TITLE
delegis:0.1.0

### PR DESCRIPTION
**Pull request by pklaschka**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: A package and template for creating legislative documents (such as bylaws for non-profits) using Typst. While the template is focused on German documents (such as a _Satzung_ and _Vereinsordnungen_ for non-profits, which is also the basis of this template: the Satzung of WüSpace e. V.), any strings used in the template can be customized, making it possible to use this in other languages, as well.

Note that the package is maintained by WüSpace e. V. (https://github.com/wuespace), and not necessarily myself individually.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/delegis/0.1.0/template/main.typ [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#import "@preview/delegis:0.1.0": *
+
+#show: it => delegis(
+  // Metadata
+  title: "Vereinsordnung zu ABCDEF",
+  abbrev: "ABCDEFVO",
+  resolution: "3. Beschluss des Vorstands vom 24.01.2024",
+  inEffect: "24.01.2024",
```
Comment (laurmaedje) [inline packages/preview/delegis/0.1.0/template/main.typ:8]: while not strictly required, I would suggest using `kebab-case` for variable and argument names as it is the idiomatic Typst case convention

Comment (pklaschka) [reply packages/preview/delegis/0.1.0/template/main.typ:8]: Ah, true. I'll fix that (I worked too much in other languages this week, still working on getting back into the Typst way of things).

**Comment by pklaschka** [comment]
(I've adjusted it to kebab-case and added a bit more documentation, so this should be ready for review again. Thanks!)

File: packages/preview/delegis/0.1.0/template/main.typ [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#import "@preview/delegis:0.1.0": *
+
+#show: it => delegis(
+  // Metadata
+  title: "Vereinsordnung zu ABCDEF",
+  abbrev: "ABCDEFVO",
```
Comment (laurmaedje) [inline packages/preview/delegis/0.1.0/template/main.typ:6]:
```suggestion
  abbreviation: "ABCDEFVO",
```

Comment (pklaschka) [reply packages/preview/delegis/0.1.0/template/main.typ]: Good catch – thanks! Sorry, that one slipped through the cracks.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**